sdk/storage/storage-queue/src/RetryPolicyFactory.ts

```diff
@@ -2,24 +2,30 @@
 // Licensed under the MIT License.
 
 import { RequestPolicy, RequestPolicyFactory, RequestPolicyOptions } from "@azure/core-http";
-import { RetryPolicy, RetryPolicyType } from "./policies/RetryPolicy";
+import {
+  RetryPolicy as StorageQueueRetryPolicy,
+  RetryPolicyType as StorageQueueRetryPolicyType
+} from "./policies/RetryPolicy";
 
-export { RetryPolicyType } from "./policies/RetryPolicy";
+export {
+  RetryPolicyType as StorageQueueRetryPolicyType,
+  RetryPolicy as StorageQueueRetryPolicy
+} from "./policies/RetryPolicy";
 
 /**
  * Retry options interface.
  *
  * @export
  * @interface RetryOptions
  */
-export interface RetryOptions {
+export interface StorageQueueRetryOptions {
   /**
    * Optional. RetryPolicyType, default is exponential retry policy.
    *
    * @type {RetryPolicyType}
    * @memberof RetryOptions
```

Comment:
Update all instances of the `@memberof` to use the new name

Reply:
Will do. Caught this in the storage-blob one, you reviewed too quickly for me to fix this here :)